tools/.codespell_whitelist

```diff
@@ -14,6 +14,7 @@ nknown
 od
 ot
 process'
+referer
```

Comment:
I'd actually like to keep this off the list :P It's a misspelling that's become "official" by virtue of it being widely used, but I'd like to use referrer elsewhere at least for now. Maybe the misspelling will eventually become the accepted spelling...

Reply:
Yes I'll drop those on next rebase ;)